Libraries/Modal/Modal.js

```diff
@@ -64,68 +69,59 @@ export type Props = $ReadOnly<{|
   ),
 
   /**
-   * The `transparent` prop determines whether your modal will fill the
-   * entire view.
-   *
-   * See https://reactnative.dev/docs/modal.html#transparent
+    The `transparent` prop determines whether your modal will fill the entire view.
+    Setting this to `true` will render the modal over a transparent background.
    */
   transparent?: ?boolean,
 
   /**
-   * The `statusBarTranslucent` prop determines whether your modal should go under
-   * the system statusbar.
-   *
-   * See https://reactnative.dev/docs/modal.html#transparent
+    The `statusBarTranslucent` prop determines whether your modal should go under the system statusbar.
+
+    @platform android
    */
   statusBarTranslucent?: ?boolean,
 
   /**
-   * The `hardwareAccelerated` prop controls whether to force hardware
-   * acceleration for the underlying window.
-   *
-   * This prop works only on Android.
-   *
-   * See https://reactnative.dev/docs/modal.html#hardwareaccelerated
+    The `hardwareAccelerated` prop controls whether to force hardware acceleration for the underlying window.
+
+    @platform android
    */
   hardwareAccelerated?: ?boolean,
 
   /**
-   * The `visible` prop determines whether your modal is visible.
-   *
-   * See https://reactnative.dev/docs/modal.html#visible
+    The `visible` prop determines whether your modal is visible.
    */
   visible?: ?boolean,
 
   /**
-   * The `onRequestClose` callback is called when the user taps the hardware
-   * back button on Android or the menu button on Apple TV.
-   *
-   * This is required on Apple TV and Android.
-   *
-   * See https://reactnative.dev/docs/modal.html#onrequestclose
+    The `onRequestClose` callback is called when the user taps the hardware back button on Android
+    or the menu button on Apple TV. Because of this required prop, be aware that `BackHandler`
+    events will not be emitted as long as the modal is open.
+
+    @type {@platform android, Platform.isTVOS}{@required yes} function
+    @type {@platform (others)}{@required no} function
```

Comment:
# Point of discussion

The required parameter changes with the platform used.

## Screenshot
![Screenshot from 2020-06-22 16-36-59](https://user-images.githubusercontent.com/33171576/85287512-ffad9b00-b4b1-11ea-9b14-0c27d7cf3fcf.png)